config/config.py

```diff
@@ -264,12 +293,82 @@ def build_config(era, sample):
         [[TauPtCorrection, "global"]],
         sanetize_producers=[[LVMu1, "mt"], [VetoMuons, "mt"]],
     )
-    # # Jet energy resolution
-    # shift_dict = {"JE_reso_shift": 1}
-    # AddSystematicShift(config, "jerUncUp", {"global": shift_dict}, [[JetEnergyCorrection, "global"]])
-    # shift_dict = {"JE_reso_shift": -1}
-    # AddSystematicShift(config, "jerUncDown", {"global": shift_dict}, [[JetEnergyCorrection, "global"]])
-    # # Jet energy scale
+    # Add MET shifts
+    AddSystematicShift(
```

Comment:
This needs a separate helper function (see comment above) that shifts the nanoAOD quantity (and not the producer which is done here without necessity)

Reply:
So you would recommend to remove the `register_external_shift` function from the NanoAODQuantities definition file and instead use a new helper function to register these shifts for the quantities ?